docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -0,0 +1,468 @@
+# Credit Class, Project, and Batch Management
+
+This tutorial covers the creation and management of credit classes, projects, and batches using the [command-line interface (CLI)](../../ledger/infrastructure/interfaces.html#command-line-interface). This tutorial will demonstrate with data standards and practices used by Regen Network Development for [Regen Registry Program](https://library.regen.network/).
+
+For information about creating and managing credit classes, projects, and credit batches using the [Regen Marketplace](https://app.regen.network/) application, see [Regen Network Guidebook](https://guides.regen.network/guides/regen-marketplace).
+
+## Prerequisites
+
+- [Install Regen](../../ledger/get-started)
+- [Redwood Testnet](../../ledger/get-started/live-networks.md#redwood-testnet)
+
+## Recommended
+
+- [Data Module Overview](../../modules/data)
+- [Data Module Concepts](../../modules/data/01_concepts.md)
+- [Ecocredit Module Overview](../../modules/ecocredit)
+- [Ecocredit Module Concepts](../../modules/ecocredit/01_concepts.md)
+
+## Metadata
+
+To create classes, projects, and batches on chain using the `ecocredit` module, we first need to know what our supporting data looks like for each class, project, and batch we intend to create.
+
+Credit classes, projects, and batches are stored as objects in on-chain application state. Each object has a `metadata` field that can be any arbitrary `string` with a maximum length of `256`. There are no additional restrictions on `metadata` but it was designed for content hashes.
```

Comment:
Some users might not use an identifier. I think content hash here makes sense.

Reply:
I think we mean different things by identifiers. In my understanding, this addition just adds some description to how content hashes & IRIs in our system are used as identifiers.

Reply:
'designed for content hashes so that it can be used for an identifier' seems a bit confusing and unnecessary given the next paragraph explains how RND uses an IRI that includes a content hash.

Reply:
I didn't make any changes here as it seems redundant with the next paragraph.